src/kinby/plugins/skills.py

```
@@ -0,0 +1,116 @@
+"""Load skill instructions and expose them to one model turn."""
+
+from __future__ import annotations
+
+from collections.abc import Sequence
+from dataclasses import dataclass
+from pathlib import Path
+from typing import NewType
+
+from kinby.contracts import Warning
```

Ruff (0.16.2)

[error] Import `Warning` is shadowing a Python builtin

(A004)